src/mod/asr_tts/mod_unimrcp/mod_unimrcp.c

```diff
@@ -357,6 +357,8 @@ struct speech_channel {
 	audio_queue_t *audio_queue;
 	/** True, if channel was opened successfully */
 	int channel_opened;
+	/** channel destroy flag*/
+	apt_bool_t channel_destroyed;
```

Comment:
instead of using apt_bool_t, we can replace this with an integer similar to channel_opened variable. Just to be consistent in the code implementation of the author.